src/workerd/api/http.c++

```diff
@@ -2014,6 +2014,42 @@ jsg::Promise<void> Fetcher::delete_(
       js, JSG_THIS, kj::mv(url), kj::mv(subInit), featureFlags));
 }
 
+jsg::Promise<QueueResponse> Fetcher::queue(
+      jsg::Lock& js, kj::String queueName, kj::Array<ServiceBindingQueueMessage> messages) {
+  auto& ioContext = IoContext::current();
+  auto worker = getClient(ioContext, nullptr, "queue"_kj);
+
+  auto encodedMessages = kj::heapArrayBuilder<IncomingQueueMessage>(messages.size());
+  for (auto& msg : messages) {
+    jsg::Serializer serializer(js.v8Isolate, jsg::Serializer::Options {
+        .version = 15,
+        .omitHeader = false,
+    });
```

Comment:
note for later: We might want to rethink the `jsg::Serializer` API just a bit such that a single instance can be reset/reused after calling `release()` so that we can avoid cases like this where we create a new instance on every iteration. Extremely minor tho.